sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -56,15 +56,16 @@ az cognitiveservices account create \
     --yes
 ```
 ### Authenticate the client
-In order to interact with the Text Analytics service, you'll need to create an instance of the [TextAnalyticsClient](#create-a-client) class. You would need an **endpoint** and **subscription key** to instantiate a client object.
+In order to interact with the Text Analytics service, you'll need to create an instance of the 
+`TextAnalyticsClient` class. You would need an **endpoint** and **API key** to instantiate a client object.
```

Comment:
How can an user find the endpoint? We could mention to get it from the Azure CLI or the Azure portal

Reply:
updated to 
```
In order to interact with the Text Analytics service, you'll need to create an instance of the 
`TextAnalyticsClient` class. You would need an **endpoint** and **API key** to instantiate a client object.
And they can be found in the [Azure Portal][azure_portal] under the "Quickstart" in your created
Text Analytics resource. 
```